src/Kestrel.Core/Internal/Infrastructure/HttpUtilities.cs

```diff
@@ -135,17 +136,17 @@ public static string GetAsciiStringEscaped(this Span<byte> span, int maxChars)
         /// </remarks>
         /// <returns><c>true</c> if the input matches a known string, <c>false</c> otherwise.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static unsafe bool GetKnownMethod(this Span<byte> span, out HttpMethod method, out int length)
+        public static unsafe bool TryGetKnownMethod(this Span<byte> span, out HttpMethod method, out int length)
         {
             fixed (byte* data = &MemoryMarshal.GetReference(span))
             {
-                method = GetKnownMethod(data, span.Length, out length);
+                method = TryGetKnownMethod(data, span.Length, out length);
                 return method != HttpMethod.Custom;
             }
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal static unsafe HttpMethod GetKnownMethod(byte* data, int length, out int methodLength)
+        internal static unsafe HttpMethod TryGetKnownMethod(byte* data, int length, out int methodLength)
```

Comment:
This shouldn't be Try

Reply:
Because it always returns a value in `out`?

Reverted